src/EnergyPlus/SizingManager.cc

```diff
@@ -1857,170 +1852,171 @@ void ManageSystemVentilationAdjustments(EnergyPlusData &state)
                                                                      state.dataOutRptPredefined->pdchS62zcdBox,
-                                                                     TermUnitFinalZoneSizing(termUnitSizingIndex).ZoneName,
+                                                                     thisTermUnitFinalZoneSizing.ZoneName,
                                                                      AirDistUnit(iAirDistUnit).EquipType(1)); // use first type of equipment listed
                             break; // if it has been found no more searching is needed
                         }
                     }
                     OutputReportPredefined::PreDefTableEntry(state,
                                                              state.dataOutRptPredefined->pdchS62zcdVpz,
-                                                             TermUnitFinalZoneSizing(termUnitSizingIndex).ZoneName,
+                                                             thisTermUnitFinalZoneSizing.ZoneName,
                                                              state.dataSize->VpzClgByZone(termUnitSizingIndex),
                                                              4); // Vpz LS:
                     OutputReportPredefined::PreDefTableEntry(state,
                                                              state.dataOutRptPredefined->pdchS62zcdVdz,
-                                                             TermUnitFinalZoneSizing(termUnitSizingIndex).ZoneName,
+                                                             thisTermUnitFinalZoneSizing.ZoneName,
                                                              state.dataSize->VdzClgByZone(termUnitSizingIndex),
                                                              4); // Vdz
                     OutputReportPredefined::PreDefTableEntry(state,
                                                              state.dataOutRptPredefined->pdchS62zcdVpzmin,
-                                                             TermUnitFinalZoneSizing(termUnitSizingIndex).ZoneName,
+                                                             thisTermUnitFinalZoneSizing.ZoneName,
                                                              state.dataSize->VpzMinClgByZone(termUnitSizingIndex),
                                                              4); // Vpz-min
                     // Vpz-min, simplified procedure?
-                    if (TermUnitFinalZoneSizing(termUnitSizingIndex).VpzMinByZoneSPSized) {
+                    if (thisTermUnitFinalZoneSizing.VpzMinByZoneSPSized) {
                         OutputReportPredefined::PreDefTableEntry(
-                            state, state.dataOutRptPredefined->pdchS62zcdVpzminSPSize, TermUnitFinalZoneSizing(termUnitSizingIndex).ZoneName, "Yes");
+                            state, state.dataOutRptPredefined->pdchS62zcdVpzminSPSize, thisTermUnitFinalZoneSizing.ZoneName, "Yes");
```

Comment:
Was the name `pdchS62zcdVpzminSPSize` created using an automatic password generator?